src/react-components/room/PromoteClientModal.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FormattedMessage } from "react-intl";
+import { Modal } from "../modal/Modal";
+import styles from "./PromoteClientModal.scss";
+import { Button } from "../input/Button";
+import { CloseButton } from "../input/CloseButton";
+
+export function PromoteClientModal({ onClose, onConfirm, displayName }) {
+  return (
+    <Modal title="Promote User" beforeTitle={<CloseButton onClick={onClose} />} contentClassName={styles.modalContent}>
+      <p>
+        <FormattedMessage id="promote.message" />
+      </p>
+      <Button preset="accept" onClick={onConfirm}>
+        <FormattedMessage id="promote.confirm-prefix" />
```

Comment:
Something odd is happening for me with the whitespace here. The `displayName` `span` is pushed right up against, `"Yes, Promote"`,
![image](https://user-images.githubusercontent.com/4072106/98055514-78265000-1df2-11eb-8a4d-3a91eac8b489.png)
but then if I add a `&nbsp;`, the whitespace between `"Promote"` and the username is too big:
![image](https://user-images.githubusercontent.com/4072106/98055470-60e76280-1df2-11eb-9e10-028c45a564fb.png)
```jsx
        <FormattedMessage id="promote.confirm-prefix" />&nbsp;
```